banks.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo "received args: $@"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)